README.md

```diff
@@ -120,7 +120,7 @@ defined in test suite files.
       --strict                 strict parse the testsuites (default false)
   -v, --values stringArray     absolute or glob paths of values files location, default no values files
   -f, --file stringArray       glob paths of test files location, default to tests\*_test.yaml (default [tests\*_test.yaml])
-  -3, --helm3                  parse helm charts as helm3 charts (default false)
```

Comment:
Is there a way to just ignore the old/deprecated flags, to limit the impact of breaking change for people already using helm 3? Otherwise everyone will probably get "unknown shorthand flag: '3' in -3" :/

Reply:
I can keepreading old flug, but just ignore it value when executing the command. Wdyt ?

Reply:
@rquinio i appliedthe changed plz check